### PR TITLE
Mention server support restrictions, fix #59

### LINKED ISSUE #59: Compatibility with websockets-pubsub
Some Solid pod servers (like ESS) implement websockets-notification-servers.
Others (like NSS, PSS, CSS, ...) implement websockets-pubsub.

Will this library support both protocols or will it only specialise in websockets-notification-servers?

### PR DESCRIPTION
**Pull request by michielbdejong**
CC @nicolasmondada

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,12 @@ like [Babel](https://babeljs.io), and to add polyfills for e.g. `Map`, `Set`,
 `Promise`, `Headers`, `Array.prototype.includes`, `Object.entries` and
 `String.prototype.endsWith`.
 
+# Server support
+Unfortunately this client only works with Inrupt's ESS server, not with
+other Solid servers like NSS, PSS, CSS, TrinPod, and DexPod. We are looking
+at different options to resolve this in a next version; see
```
Comment (nicolasmondada) [inline README.md:24]:
```suggestion
This client only works with Inrupt's ESS, providing secure notifications over Websockets. Unfortunately, this is not compatible with websockets notifications implemented by other Solid servers like NSS, PSS, CSS, TrinPod, and DexPod. These servers websocket notifications they are based on [Solid Websockets API](https://github.com/solid/solid-spec/blob/master/api-websockets.md), a specification not recommended for secure messaging. We are looking at different options to help developers get their apps working securely across all Solid servers.
```

**Review by nicolasmondada — CHANGES_REQUESTED** [review]
I adjusted the text to explain a bit more about the decision behind the current incompatibility.

**Comment by ThisIsMissEm** [comment]
@kay-kim @nicolasmondada do we want to accept this and add it to the main page of the documentation as an admonition much like what I did with access-grants? (I assume this is still relevant today)

**Comment by ThisIsMissEm** [comment]
Closing this in favour of changes in #244 — note: this updates the docs, but leaves the readme as-is; The documentation is the source of truth, not the README

**Pull request closed without merging**